mods/d2k/rules/misc.yaml

```diff
@@ -102,10 +102,6 @@ crate:
 		SelectionShares: 0
 		NoBaseSelectionShares: 9001
 		Units: mcv
-	GrantUpgradeCrateAction@cloak:
-		SelectionShares: 5
-		Effect: cloak
-		Upgrades: cloak
```

Comment:
You may also fix it the other way round, because this was added on purpose in https://github.com/OpenRA/OpenRA/pull/6856. It also proofs my point about the weak spot about https://github.com/OpenRA/OpenRA/wiki/Actor-Upgrades. They are easy to do wrong without anyone noticing. This lint checks are long overdue. Thanks!